adhocteam/Head-Start-TTADP
# [TTAHUB-314] Remove included model that changed counts

## Description of change
Discovered via testing that the raw:true attribute seemed to add an unpredictable number to the count returned for the Topic/Frequency graph. I'm not sure why that would be. Would love any available insight.

edit: here is a stack overflow answer with some theorizing
https://stackoverflow.com/questions/64546830/sequelize-how-to-eager-load-with-associations-raw-true

and here is my recorded observations
https://gist.github.com/thewatermethod/ab2df41b354269993a112983eba76ac3

I verified the correct number via direct SQL queries and proceeded from there.

Removed the attribute, which then necessitated moving the role filtering from the frontend to the backend.

## How to test
Visit the regional dashboard. View the topic/frequency graph and check the numbers against CSV export or your local database. Likewise, check with the roles filter.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-314


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
